recruiter/models.py

```diff
@@ -35,3 +35,13 @@ class Recruiter(models.Model):
 
     def __str__(self):
         return f'{self.name}'
+
+    @classmethod
+    def is_recruiter(cls, auth_user_id):
+        """
+        For checking if a currently logged in user is a Recruiter
+        """
+        user = cls.objects.filter(user_id=auth_user_id)
```

Comment:
blank line between docstring and the code

Reply:
Will fix

Reply:
Umm I don't think there's a blank line it just looks like that (you can see the line numbers)